documentation/homework/week1/todo-app/src/components/todo-item.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+export default class todoItem extends Component {
+  render() {
+    const { description, deadLineDate, isDone} = {
+      description: this.props.description,
+      deadLineDate: new Date(this.props.deadLineDate).toDateString(),
+      _isDone: this.props.isDone,
+      get isDone() {
```

Comment:
Again, not really sure what's happening here. I would suggest something like:

    const { isDone } = this.props
    if (isDone()) {
        ...whatever you want to do

Reply:
yeah sure next time 💔